appium/README.md

```diff
@@ -25,13 +25,16 @@ To run a particular test:
 
 To run all tests: `npm run-script test.live.all` or `npm run-script test.mock.all`
 
+## Building app for testing
+For debugging purposes instead of running `bundle exec fastlane build` you can copy `FlowCrypt.app` from `/DerivedData/FlowCrypt-.../Build/Products/Debug-iphonesimulator` (In Xcode open `Products` folder -> `FlowCrypt` -> `Show in Finder`) to appium folder. 
+
```

Comment:
Please pay attention to the whole document holistically.. documentation must be as clean and orderly as code, if not more so.